### DISCUSSION TITLE
What is the default output for Svelte-Kit?

### DISCUSSION THREAD

**Discussion by melMass** [vercel/community#894] [Help] [answered]
Hi,

I need to customise the build command of my Svelte Kit project which builds ok but complains about my output dir.
The greyed out value of the template states `dist` before unlocking it, but that's not what the default value actually is apparently (as it works using the default value but there is not `dist` output)

I also asked the question on Svelte's discord so I'll answer myself here if I get an answer there!

Thanks

**Comment by swarnava** [comment, marked as answer]
>  The greyed out value of the template states dist before unlocking it, but that's not what the default value actually is apparently (as it works using the default value but there is not dist output)

Could you elaborate this bit more? Perhaps a screenshot would be helpful

**Reply by melMass** [reply]
I'm referring to the build templates on vercel:
<img width="947" alt="Screenshot 2022-09-27 at 18 27 32" src="https://user-images.githubusercontent.com/7041726/192582609-47b014f6-55cb-4e88-ad3d-7965681a31b9.png">

I did fix it by using the root directory as my monorepo sub app and setting the build options as in the screenshot.

Since then I gathered a few more informations on the issue, sveltekit uses [Vercel's Build API V3](https://vercel.com/docs/build-output-api/v3) (https://github.com/sveltejs/kit/pull/4663).

But even after looking through the code and doc I don't think there is any folder you can manually point it to for now.

Related issue: https://github.com/sveltejs/kit/issues/4755

**Reply by amyegan** [reply]
Further clarification requested in https://github.com/vercel/community/discussions/904
